src/databricks/labs/lsql/dashboards.py

```diff
@@ -897,32 +897,32 @@ def _from_dashboard_folder(cls, folder: Path) -> "DashboardMetadata":
             tiles.append(tile)
         return cls(display_name=folder.name, _tiles=tiles)
 
-    def export_to_zipped_csv(self, sql_backend: SqlBackend, export_path: Path) -> Path:
+    def export_to_zipped_csv(self, sql_backend: SqlBackend, export_path: Path, zip_file_name: str) -> Path:
```

Comment:
```suggestion
    def export_to_zipped_csv(self, sql_backend: SqlBackend, export_path: Path) -> Path:
```
Expect the zip file to be part of the export path

Reply:
@JCZuurmond 

Are you okay with 
        `if not export_path.suffix == '.zip':
            raise ValueError("The export_path must point to a .zip file.")`
To capture any path not pointing to a .zip path?

Reply:
Or 
`        if not export_path.suffix == '.zip':
            logger.error("The export_path must point to a .zip file.")`

Reply:
@JCZuurmond 
Removed extra  suffix validation on the export_to_zipped_csv function.